### PR TITLE
Fix a typo in the Distributed training guide

### PR DESCRIPTION
**Pull request by synandi**
Fixed typos in lines 133 &860.

### CODE REVIEW

**Review by Anuragkillswitch — DISMISSED** [review]

File: site/en/guide/distributed_training.ipynb [hunk 1]
```diff
@@ -857,7 +857,7 @@
...
         "There are two components of `'TF_CONFIG'`: a cluster and a task.\n",
         "\n",
-        "- A cluster provides information about the training cluster, which is a dict consisting of different types of jobs such as workers. In multi-worker training, there is usually one worker that takes on a little more responsibility like saving checkpoint and writing summary file for TensorBoard in addition to what a regular worker does. Such worker is referred to as the \"chief\" worker, and it is customary that the worker with index `0` is appointed as the chief worker (in fact this is how `tf.distribute.Strategy` is implemented).\n",
+        "- A cluster provides information about the training cluster, which is a dict consisting of different types of jobs such as workers. In multi-worker training, there is usually one worker that takes on a little more responsibility like saving checkpoint and writing summary file for TensorBoard in addition to what a regular worker does. Such a worker is referred to as the \"chief\" worker, and it is customary that the worker with index `0` is appointed as the chief worker (in fact this is how `tf.distribute.Strategy` is implemented).\n",
```
Comment (8bitmp3) [inline site/en/guide/distributed_training.ipynb:860]:
```suggestion
        "- A cluster provides information about the training cluster, which is a dict consisting of different types of jobs such as workers. In multi-worker training, there is usually one worker that takes on a little more responsibility like saving checkpoint and writing summary file for TensorBoard in addition to what a regular worker does. Such worker is referred to as the \"chief\" worker, and it is customary that the worker with index `0` is appointed as the chief worker (in fact this is how `tf.distribute.Strategy` is implemented).\n",
```

File: site/en/guide/distributed_training.ipynb [hunk 2]
```diff
@@ -130,7 +130,7 @@
         "| **Custom training loop** | Supported          | Supported     | Supported                     | Experimental support     | Experimental support      |\n",
         "| **Estimator API**        | Limited Support    | Not supported | Limited Support               | Limited Support          | Limited Support           |\n",
         "\n",
-        "Note: [Experimental support](https://www.tensorflow.org/guide/versions#what_is_not_covered) means the APIs are not covered by any compatibilities guarantees.\n",
+        "Note: [Experimental support](https://www.tensorflow.org/guide/versions#what_is_not_covered) means the APIs are not covered by any compatibily guarantees.\n",
```
Comment (8bitmp3) [inline site/en/guide/distributed_training.ipynb:133]: @synandi 

```suggestion
        "Note: [Experimental support](https://www.tensorflow.org/guide/versions#what_is_not_covered) means the APIs are not covered by any compatibility guarantees.\n",
```

@synandi Please be careful. Your fix contained a typo.

**Review by 8bitmp3 — APPROVED** [review]

**Comment by 8bitmp3** [comment]
Fixed a typo in the fix itself. LGTM.

File: site/en/guide/distributed_training.ipynb (hunk 2 above)
Comment (synandi) [reply site/en/guide/distributed_training.ipynb]: @8bitmp3  acknowledged.

**Pull request merged**